tools/search_files.py

```diff
@@ -8,15 +8,12 @@
 import argparse
 import utils
 
-def main():
+def DoMain(args = []):
   parser = argparse.ArgumentParser(
     description='Search files with a specific extension under a directory',
     fromfile_prefix_chars='@'
   )
   parser.add_argument('--ext', required=True, help='extension to search for')
   parser.add_argument('directory', help='input directory')
-  options = parser.parse_args()
-  print('\n'.join(utils.SearchFiles(options.directory, options.ext)))
-
-if __name__ == "__main__":
-  main()
+  options = parser.parse_args(args)
+  return '\n'.join(utils.SearchFiles(options.directory, options.ext))
```

Comment:
```suggestion
  return '\n'.join(utils.SearchFiles(options.directory, options.ext))

if __name__ == '__main__':
  DoMain(sys.argv[1:])
```

To allow running the script manually?